packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -510,36 +512,68 @@ function SettingsSection( {
 	);
 }
 
-function DataviewsViewConfigContent( {
+function DataviewsViewConfigDropdown( {
 	density,
 	setDensity,
 }: {
 	density: number;
 	setDensity: React.Dispatch< React.SetStateAction< number > >;
 } ) {
 	const { view } = useContext( DataViewsContext );
+	const popoverId = useInstanceId(
+		_DataViewsViewConfig,
+		'dataviews-view-config-dropdown'
+	);
+	const popoverProps = useMemo(
+		() => ( {
+			...DATAVIEWS_CONFIG_POPOVER_PROPS,
+			id: popoverId,
+		} ),
+		[ popoverId ]
+	);
```

Comment:
Is the memoization of the popover props necessary? We'll always merge them anyway:

https://github.com/WordPress/gutenberg/blob/2c62c6aa2f30060b3acd40b225aebe4295ad0bf4/packages/components/src/dropdown-menu/index.tsx#L85-L91